Core/CategoryDTO.cs

```diff
@@ -1,25 +1,16 @@
-﻿namespace Core
+﻿using System.Collections.Generic;
+
+namespace Core
 {
     public class CategoryDto
     {
-		public CategoryDto(string name, string cardNumber, string applicationCode, string isIncome, 
-			string isExpence, string terminal, string description)
+		public CategoryDto(string name, List<Rule> rules)
 		{
 			Name = name;
-			CardNumber = cardNumber;
-			ApplicationCode = applicationCode;
-			IsIncome = isIncome;
-			IsExpence = isExpence;
-			Terminal = terminal;
-			Description = description;
+			Rules = rules;
 		}
 
-		public string Name { get; }
-		public string CardNumber { get; }
-		public string ApplicationCode { get; }
-		public string IsIncome { get; }
-		public string IsExpence { get; }
-		public string Terminal { get; }
-		public string Description { get; }
+		public string Name { get; set; }
+		public List<Rule> Rules { get; set; }
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make the 'Rules' property read-only by removing the property setter or making it private.](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=4970545)